rest_framework/compat.py

```diff
@@ -161,10 +161,16 @@ def is_guardian_installed():
 try:
     import markdown
 
-    if markdown.version <= '2.2':
+    if hasattr(markdown, 'version'):
```

Comment:
Shouldn't have been better to do a "try markdown_version=markdown.version ... except AttributeError"?